php/php.project/src/org/netbeans/modules/php/project/ui/actions/tests/GoToTest.java

```diff
@@ -54,6 +55,14 @@
     private static final Logger LOGGER = Logger.getLogger(GoToTest.class.getName());
     private static final RequestProcessor RP = new RequestProcessor(GoToTest.class.getName(), 2);
 
+    private static class FileObjectComparator implements Comparator<FileObject> {
```

Comment:
Just a note: I usually put the inner classes at the very end of the class (usually below `//~ Inner classes` comment). No need to change it in this PR, I just think it makes the code more readable (main class and then inner classes, not mixed).